code/__HELPERS/matrices.dm

```diff
@@ -60,30 +96,14 @@
 	. += f
 	. += 1
 
-//The X pixel offset of this matrix
+///The X pixel offset of this matrix
 /matrix/proc/get_x_shift()
 	. = c
 
-//The Y pixel offset of this matrix
+///The Y pixel offset of this matrix
 /matrix/proc/get_y_shift()
 	. = f
 
-/matrix/proc/get_x_skew()
-	. = b
-
-/matrix/proc/get_y_skew()
-	. = d
-
-//Skews a matrix in a particular direction
-//Missing arguments are treated as no skew in that direction
-
-//As Rotation is defined as a scale+skew, these procs will break any existing rotation
-//Unless the result is multiplied against the current matrix
-/matrix/proc/set_skew(x = 0, y = 0)
-	b = x
-	d = y
-
-
```

Comment:
This seems like a very abritrary removal

Reply:
That's a part of the port, but I will exempt that part from the port then